test/e2e/e2e_suite_test.go

```diff
@@ -260,6 +274,29 @@ func initBootstrapCluster(bootstrapClusterProxy framework.ClusterProxy, config *
 	}
 	_, err = chart.InstallChart(nil)
 	Expect(err).ToNot(HaveOccurred())
+
+	By("Installing rancher chart")
+	chart = &operatorframework.HelmChart{
+		BinaryPath: helmBinaryPath,
+		Path:       rancherPath,
+		Name:       "rancher",
+		Kubeconfig: bootstrapClusterProxy.GetKubeconfigPath(),
+		Output:     operatorframework.Full,
+		AdditionalFlags: []string{
+			"--version", "v2.7.5",
+			"--namespace", rancherNamespace,
+			"--create-namespace",
+			"--wait",
+		},
+	}
+	_, err = chart.InstallChart(map[string]string{
+		"bootstrapPassword":         "admin",
+		"features":                  "embedded-cluster-api=false",
+		"global.cattle.psp.enabled": "false",
+		"replicas":                  "1",
+		"hostname":                  os.Getenv("RANCHER_HOSTNAME"),
```

Comment:
This will have to be resolvable in test environment, you can do something similar to this https://github.com/rancher/eks-operator/blob/master/Makefile#L112 + sslip.io

Reply:
Not entirely true. You can always manually open an ingress connection to the rancher cluster, and make something like localhost resolvable. Don’t want to overcomplicate it, it could be resolvable but it does not have to.